components/sysutils/net-snmp/net-snmp-base.p5m

```diff
@@ -407,6 +407,8 @@ file build/prototype/$(MACH64)/usr/bin/$(MACH64)/mib2c-update \
 file usr/bin/net-snmp-config path=usr/bin/net-snmp-config
 file build/prototype/$(MACH64)/usr/bin/$(MACH64)/net-snmp-config \
     path=usr/bin/$(MACH64)/net-snmp-config
+link path=usr/bin/net-snmp-config-32 target=usr/bin/net-snmp-config pkg.linted=true
```

Comment:
link is incorrect. It should have been 
link path=usr/bin/net-snmp-config-32 target=net-snmp-config 
link path=usr/bin/net-snmp-config-64 target=./$(MACH64)/net-snmp-config